utils/api/schemas/page.ts

```diff
@@ -4,11 +4,11 @@ import { extendZodWithOpenApi } from '@anatine/zod-openapi';
 import { Page } from 'server/models';
 import { layoutBlockSchema } from './layout';
 import { communitySchema } from './community';
+import { baseSchema } from '../utils/baseSchema';
 
 extendZodWithOpenApi(z);
 
-export const pageSchema = z.object({
-	id: z.string().uuid(),
+export const pageSchema = baseSchema.extend({
```

Comment:
i do this for all the schemas that represent models that are `get` or `getMany`able.

This makes is possible to query by `updatedAt` and `createdAt`

the `baseSchema` is 
```ts
const baseSchema = z.object({
    updatedAt: z.string().datetime().optional(),
    createdAt: z.string().datetime().optional(),
		id: z.string().uuid(),
})
```